src/main/scala/net/snowflake/spark/snowflake/pushdowns/querygeneration/WindowStatement.scala

```diff
@@ -26,11 +31,70 @@ private[querygeneration] object WindowStatement {
     expAttr: (Expression, Seq[Attribute])
   ): Option[SnowflakeSQLStatement] = {
     val expr = expAttr._1
+    val fields = expAttr._2
 
     Option(expr match {
-      case _: RowNumber => ConstantString(expr.prettyName.toUpperCase) + "()"
+      // Handle Window Expression. (It is moved from MiscStatement)
```

Comment:
Don't add this as a comment.

Reply:
Remove " (It is moved from MiscStatement)" for this and another place.